datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts

```diff
@@ -47,14 +71,19 @@ export function filterSchemaRows(
         entityRegistry,
         formattedFilterText,
     );
+
+    const filteredEditableDescription = getDescriptionFieldPathsByMetadata(editableSchemaMetadata, formattedFilterText);
```

Comment:
since we need to do slightly more work for checking things inside of `editableSchemaMetadata` and we're already doing that in `getFilteredFieldPathsByMetadata`, how do you feel about moving the edited description filter check inside of that function? that way we just have one checking for things in `editableSchemaMetadata`.

We could then just have one filter function for `matchesDescription` like you already have that is reused between that function and down below similar to how `matchesTagsOrTerms` is being used twice

Reply:
Worked on that and committed the changes. Please check.